Pipfile.lock

```diff
@@ -449,53 +413,6 @@
             "markers": "python_version >= '3.7'",
             "version": "==0.14.0"
         },
-        "hiredis": {
```

Comment:
I think we should keep this, probably with `redis = {extras = ["hiredis", version = "*"}` in the Pipfile